components/engine_test/src/lib.rs

```diff
@@ -205,6 +309,30 @@ pub mod kv {
             None
         }
 
+        fn open_tablet_cache_latest(&self, id: u64) -> Option<KvTestEngine> {
```

Comment:
The performance of this API is pretty bad----it requires O(N) to query all the keys to find the latest one. 
Maybe it's better to have a map <region_id, latest_suffix> for a quick search. 
Given its usage scenario is on the hot read path, the performance of this API is critical.